rlp/sedes/serializable.py

```diff
@@ -354,14 +366,20 @@ def __new__(cls, name, bases, attrs):
             in zip(meta.field_names, meta.field_attrs)
         }
 
+        removed_parent_field_props = {
+            field_name: RemovedParentField(field_name)
+            for base in bases if issubclass(base, Serializable) if hasattr(base, '_meta')
```

Comment:
I've never seen a double `if` like this. Is it equivalent to `if issubclass(base, Serializable) and hasattr(base, '_meta')`?

Reply:
Honestly, this is an accident and I'm suprised it works.

```python
>>> [i for i in range(100) if i % 3 == 0 if i % 5 ==0]
[0, 15, 30, 45, 60, 75, 90]
```

I'm going to add an `and` and wrap it in parenthesis because this is some weird voodoo magic.